.travis.yml

```diff
@@ -1,10 +1,7 @@
 language: node_js
 
 node_js:
-  - 6
-  - 7
   - 8
-  - 9
   - 10
 
```

Comment:
I think by now this should also start listing NodeJS 12, but that could be a different follow-up PR.